wrangler-transform/widgets/Wrangler-transform.json

```diff
@@ -77,6 +104,40 @@
   ],
   "emit-alerts": true,
   "emit-errors": true,
+  "filters": [
+    {
+      "name": "PreconditionValueJEXL",
+      "condition": {
+        "expression": "preconditionToggle == 'jexl'"
```

Comment:
Have we verified an upgraded pipeline in this case? How will it render the UI when the pipeline is upgraded which didn't have any value for toggle??

Reply:
Yeah I checked this and the toggle is taking the default `jexl` value and is working fine.